### PR TITLE
Update ALOHA instructions

### PR DESCRIPTION
**Pull request by jessieay**
* Just had a leave request denied. Aaron noted "Please resubmit to your
  (direct) supervisor."

### CODE REVIEW

File: _pages/qa/aloha-request-leave-vacation-sick-time.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Yes, it's preferred if you can submit it as soon as possible.
 ## How do I request time off?
 
 1. Visit [ALOHA](http://aloha.gsa.gov/) and use your GSA credentials to log in.
-2. Submit your supervisor approval to Aaron Snow.
+2. Submit your supervisor approval to your supervisor.
```
Comment (jessieay) [inline _pages/qa/aloha-request-leave-vacation-sick-time.md]: it is possible that I am misunderstanding this -- in ALOHA there is a field for indicating supervisor. I put Aaron there. Is there a separate step where we submit the ALOHA approval to Aaron? If so, how does that work, and do we need to get confirmation from him?

**Comment by jessieay** [comment]
HI @lsgitter -- can you check on whether the above change is correct?

**Comment by jessieay** [comment]
cc @melodykramer @andrewmaier

**Comment by andrewmaier** [comment]
I think this is correct given the circumstances.

**Comment by lsgitter** [comment]
yep looks good.

**Pull request merged**